docs/howto/operate/new-cluster/aws.md

```diff
@@ -1,158 +1,209 @@
 (new-cluster:aws)=
 # Add a new AWS cluster
 
-We do not currently use Terraform to deploy a new cluster in AWS.
-Instead, we use a tool [called `kops`](https://github.com/kubernetes/kops) to provision the cluster.
+We use [eksctl](https://eksctl.io/) to provision our k8s clusters on AWS.
 
-:::{admonition} We also use `eksctl`
-In addition to `kops`, we have begun using an AWS-native tool called `eksctl` to provision Kubernetes clusters on AWS.
-See [this configuration](https://github.com/2i2c-org/infrastructure/tree/master/eksctl) for an example of a hub that uses this.
-As we understand these practices better we will update this guide with more information about `eksctl`.
+:::{note}
+
+We used to use [kops](https://kops.sigs.k8s.io/) as well, for clusters
+where we wanted to minimize base running cost. We have deprecated that now, and use
+eksctl for everything.
```

Comment:
We still have the Farallon cluster with kops, but we should be moving that one to eksctl soon, so I would be OK with this note as is.